src/workloads/scale/Mixed10KThreads.yml

```diff
@@ -22,6 +23,7 @@ Keywords:
 dbname: &dbname mixed_10k
 DocumentCount: &NumDocs 10000
 CollectionCount: &NumColls 1
+Limit: &limit 100
```

Comment:
This should be 10 and used on line 77 or deleted?

Reply:
Removed, thanks.